### ISSUE TITLE
Best settings to train Transformer-XL from scratch

### ISSUE THREAD

**Issue by AndreaLK3** [kimiyoung/transformer-xl#106]
Having read of other people's attempts (like in [Issue#92](https://github.com/kimiyoung/transformer-xl/issues/92)), and the sensitivity of Transformer-XL to hyperparameters,

if anyone has tried to train a Transformer-XL Language Model on a new corpus, I would like to hear your opinions and experiences on:

what were your hyperparameters? 
Did you use any warm-up, and if so spanning how many epochs? 
From my grid-search I would say that a learning rate in [0.0001, 0.0005] is a reasonable value.
How many epochs did you set it to train for? 10, 20, 100?
